module-5.md

```diff
@@ -12,36 +12,116 @@ Planned:      1.25 hour
+* 802.11s capable (e.g. [TP-Link TL-WN722N v1](https://www.tp-link.com/us/download/TL-WN722N_V1.html)) Radio for each Raspberry Pi 3
 
 ### Format
 
+Groups (or pairs) to start, class discussion then back to groups again
 
 ### Activity
 
-- Understanding the current internet:
-  - Speed test with library internet
-  - How much data does your app use
-
-- Grafana to show bandwidth usage:
-  - Locally stream music and video and see interface bandwidth
-  - What do your metrics mean?
-
-- Physical network topology (i.e. Toronto Mesh map) and Virtual peering topology (i.e. fc00.org)
+* Review your cell phone usage
+  * Identify your apps' data usage
+  * Record a few of the top apps that took up the most data
+  * Was this data cellular, or WiFi?
+* Public WiFi Speed Test
+  * Use a speed test tool to test the speed of the public WiFi, record the result
+    - IE: Speedtest.net
+  * Is it faster or slower than the one you have at home? 
+  * Run the speed test again, record the result
+  
+* Discuss
+  * What unit is the data usage of your apps measured in?
+  * What does MB mean?
+  * What was some of the top app and how much MB did it use? 
+    - Have participants volunteer their info if they feel comfortable doing so.
+  * Was that surprising?
+  * Would you be more/less surprised if it was cellular data? What if it was WiFi data? 
+    * Are different data sources worth more?
+    * Why? What is your cell data limit?
+  
+  * What was the unit that the WiFi speed was measured in?
+  * What does Mbps mean?
+  * How does this relate to the MB from apps?
+  * Why was the speed different between tests?
+  
+* Show Grafana functioning on a node, and describe what it is
+  * Access via web interface
+  * Show how to select nodes
+  
+* Break into groups
+  * Connect to the node
+  * Access the Grafana interface and look around
+  * Stream a music file that is located on a node's local webserver and observe the metrics
+  * Stream a video file that is located on a node's local webserver and observe the metrics
+  * Observe what metrics change if you do the same from a remote node (over the mesh)
+  * Can you measure both nodes at the same time?
+
+* Discuss results, ask the following questions:
+  * Did difficulty increase measuring two nodes at the same time?
+  * What would happen if you had a citywide mesh?
+    - Ping time
+    - Throughput
+    - CPU usage
+  * Are some metrics more important than others?
+  * How do you represent a mesh network?
+    * Physical network topology (i.e. Toronto Mesh map)
+    * Virtual peering topology (i.e. fc00.org)
 
 ## 3: Discussion and reflection activity
 
-Q & A about sharing broken internet experiences.
-
-## 4: Wrap-up and week ahead
+Q & A about sharing broken Internet experiences, understanding failure points and why we measure
+
+  * Break into 3 groups.
+  * Each group is assigned a user profile
+    - Gamer
+    - Streamer (Youtube, Netflix etc)
+    - Basic Web user (Email, Web, Im.)
```

Comment:
Perhaps with occasionally business conference calls? I feel this profile may resonate with some of our audience.